### PR TITLE
Implemented caching of superseded requests in BsRequestActionWebuiInfosJob

### PR DESCRIPTION
**Pull request by ChrisBr**
as we implemented diffs between superseded requests it is necessary to warm up the cache in the backend for these diffs.
See #4741.

### CODE REVIEW

**Comment by hennevogel** [comment]
Please be more accurate in the commit message

> remote projects / packages and links

it fails only for project links.

File: src/api/app/jobs/bs_request_action_webui_infos_job.rb [hunk 1]
```diff
@@ -8,8 +8,29 @@
...
+    for_superseded(bs_request_action, source_package_names)
+    ForSource.new(
+      bs_request_action: bs_request_action,
+      source_package_names: source_package_names
+    ).perform
+  rescue DiffError, Project::UnknownObjectError, Package::UnknownObjectError
+    # pass
+  end
+
+  def for_superseded(bs_request_action, source_package_names)
```
Comment (hennevogel) [inline src/api/app/jobs/bs_request_action_webui_infos_job.rb]: :thought_balloon: private?

**Comment by ChrisBr** [comment]
@hennevogel updated

**Comment by hennevogel** [comment]
> which causes that source_access_check! in SourcePackageFinder fails for remote projects / packages and links.

*links* is very ambiguous, can mean a lot of things in OBS context. The situation is that `SourcePackageFinder` fails for package that are in remote projects and for packages that come in via project links. Right?

**Comment by ChrisBr** [comment]
@hennevogel updated, please check again!

**Pull request merged**